### PR TITLE
AAP-1939: Update proxy support docs on AAP installer

### PR DESCRIPTION
**Pull request by TarikFD**
AAP 2.1 installer docs contain outdated proxy support content that needs updating, as well as two other fixes

Jira ticket: (Main issue) https://issues.redhat.com/browse/AAP-1939, (Bug fix 1) https://issues.redhat.com/browse/AAP-1634, (Bug fix 2) https://issues.redhat.com/browse/AAP-1680

Preview link (VPN required, See **section 5** of this doc) http://file.rdu.redhat.com/kevchin/proxy_update/tmp/en-US/html-single/#assembly-configuring-proxy-support

### CODE REVIEW

**Review by shanemcd — APPROVED** [review]

**Review by tvo318 — APPROVED** [review]
I've verified the changed content matches the content from the tower docs. Good work!

**Comment by TarikFD** [comment]
Excellent. Thanks for the help @tvo318 and @shanemcd

File: downstream/modules/platform/proc-enable-proxy-support.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+[id="proc-enable-proxy-support_{context}"]
+
+= Enable proxy support
+
+To provide proxy server support, {ControllerName} handles proxied requests (such as ALB, NLB , HAProxy, Squid, Nginx and tinyproxy in front of {ControllerName}) via the *REMOTE_HOST_HEADERS* list variable in the {ControllerName} settings. By default *REMOTE_HOST_HEADERS* is set to `["REMOTE_ADDR", "REMOTE_HOST"]`.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-enable-proxy-support.adoc:6]:
```suggestion
To provide proxy server support, {ControllerName} handles proxied requests (such as ALB, NLB , HAProxy, Squid, Nginx and tinyproxy in front of {ControllerName}) via the *REMOTE_HOST_HEADERS* list variable in the {ControllerName} settings. By default, *REMOTE_HOST_HEADERS* is set to `["REMOTE_ADDR", "REMOTE_HOST"]`.
```

File: downstream/modules/platform/proc-configure-known-proxies.adoc [hunk 2]
```diff
@@ -3,34 +3,29 @@
 
 = Configuring known proxies
 
-
 [role="_abstract"]
-You can configure a list of "known proxies" that are allowed using the `PROXY_IP_WHITELIST` setting via the settings API. Load balancers and hosts that are not on the list will result in a rejected request.
+
+To configure a list of known proxies for your {ControllerName}, add the proxy ip addresses to the *PROXY_IP_ALLOWED_LIST* field in the settings page for your {ControllerName}.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-configure-known-proxies.adoc:8]:
```suggestion
To configure a list of known proxies for your {ControllerName}, add the proxy IP addresses to the *PROXY_IP_ALLOWED_LIST* field in the settings page for your {ControllerName}.
```

**Review by ariordan-redhat — APPROVED** [review]
Added 2 very minor suggestions. LGTM.

**Pull request merged**